lib/stack/trackLoginPoints.tsx

```diff
@@ -0,0 +1,15 @@
+import { Address } from 'viem'
+import getStackClient from './getStackClient'
+import { FIRST_SMART_WALLET_LOGIN_EVENT } from '../consts'
+
+const trackLoginPoints = async (address) => {
+  const stackClient = getStackClient()
+
+  await stackClient.track(FIRST_SMART_WALLET_LOGIN_EVENT, {
+    points: 10,
```

Comment:
I'm a sucker for `angel numbers` 👼 
- last revision, I promise
```suggestion
    points: 11,
```